### ISSUE TITLE
Remember the user's choice of dark or light mode

### ISSUE THREAD

**Issue by ljflores** [ceph/pulpito-ng#57]
The current logic seems to be (correct me if I'm wrong) dark mode by default. To streamline this option, and if it's not too much overhead, it might be good to have the interface remember which setting the user chose last time they logged in.

I seem to remember the concept of user profiles being introduced in the pulpito-ng demo? This could perhaps be a setting associated with a user's profile.

**Comment by kamoltat** [comment]
Yep I thanks for pointing this out, we can add this when we integrate the auth feature into production, just only need to figure out how exactly it should look like when we save the user's choice, probably somewhere in the metadata recording user login/logout timestamp

**Comment by dikwickley** [comment]
@kamoltat how about saving the the choice in browser's local storage or even cookies.
We can even keep another option to keep it as the system's default.

If this sounds good I would like to work on it (or any other way you might suggest)

**Comment by Mamatha1718** [comment]
Hi @ljflores , @kamoltat , @dikwickley .I would like to tackle this issue, add toggle to the main src file. The issue is resolved. The PR is ready for review.
will you please let me know if any other changes required.
Thank you.
